docusaurus/docs/tutorials/ui-extensions/exposed-components.md

```diff
@@ -0,0 +1,77 @@
+---
+id: exposed-components
+title: Exposed components
```

Comment:
```suggestion
title: Expose a component
```

Reply:
We should probably split into exposing and using

Reply:
Fixed in https://github.com/grafana/plugin-tools/pull/954/commits/999f5a2263de23d9f7fd41e5f47d07843fc71ed7.